app/params/config.go

```diff
@@ -43,8 +43,8 @@ func DefaultConfig() (string, interface{}) {
 	customConfig := CustomAppConfig{
 		Config: *serverConfig,
 		WASM: WasmConfig{
-			SimulationGasLimit: 50000000,
-			QueryGasLimit:      50000000,
+			SimulationGasLimit: 5000000,
```

Comment:
We could set this to nil to use the block gas limit:

```go
// WasmConfig is the extra config required for wasm
type WasmConfig struct {
	// SimulationGasLimit is the max gas to be used in a tx simulation call.
	// When not set the consensus max block gas is used instead
	SimulationGasLimit *uint64
```

Reply:
same as consensus sounds good to me.